src/driver.jl

```diff
@@ -317,6 +317,13 @@ const __llvm_initialized = Ref(false)
     end
 
     @timeit_debug to "IR post-processing" begin
+        # mark the entry-point function (optimization may need it)
+        if deferred_codegen
+            # IDEA: save other parts of the CompileJob (so that we can reconstruct it
+            #       instead of setting it globally, which is incompatible with threading)?
```

Comment:
Yeah I am currently having LLVM handle thread saftyness https://github.com/EnzymeAD/Enzyme.jl/blob/3075eca6fb836032ae78d2c80ee02aa5f7010f48/src/compiler/orcv2.jl#L127

But these accesses are not safe https://github.com/EnzymeAD/Enzyme.jl/blob/3075eca6fb836032ae78d2c80ee02aa5f7010f48/src/compiler.jl#L4478